modules/atomic-widgets/props-resolver/transformers/styles/background-image-overlay-transformer.php

```diff
@@ -26,7 +26,7 @@ public function transform( $value, $key ) {
 			$background_style .= ' ' . $position_and_size_style;
 		}
 
-		return $background_style;
+		return $background_style . ( $value['attachment'] ? ' ' . $value['attachment'] : '' );
```

Comment:
Please use the same order as we have used in the packages PR.

I believe attachment was before position and style.